front/src/common/redux/reducers/loginReducer.js

```diff
@@ -0,0 +1,29 @@
+import * as types from '../actions/types';
+
+const initialState = {
+  isLogin: false,
```

Comment:
Please, rename it to `isLoggedIn`. It is much simpler to understand

Reply:
done